persistence/migrations/1_initial.up.sql

```diff
@@ -29,7 +29,9 @@ CREATE TABLE invoices
 
         "final_cltv_delta"      INTEGER         NOT NULL CHECK (final_cltv_delta > 0),
         "payment_addr"          BYTEA           NOT NULL UNIQUE CHECK (LENGTH(payment_addr) = 32),
-        "payment_request"       TEXT            NOT NULL CHECK (LENGTH(payment_request) > 0)
+        "payment_request"       TEXT            NOT NULL CHECK (LENGTH(payment_request) > 0),
+
+        "auto_settle"           BOOLEAN         NOT NULL
```

Comment:
why aren't you creating a new migration?

Reply:
pre: missing down.sql migration

Reply:
I think the probability of this code being used in production is very low, so I figured we could forego the migrations for now until a bit more testing has been done.